rfc6962-bis.xml

```diff
@@ -1177,6 +1177,9 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
         </list>
       </t>
+	  <t>
+        Additionally, a TLS server which supports presenting SCTs using an OCSP response MAY provide it when the TLS client included the <spanx style="verb">status_request_v2</spanx> extension (<xref target='RFC6961'/>) in the (extended) <spanx style="verb">ClientHello</spanx>, but only in addition to the <spanx style="verb">status_request</spanx> extension mentioned above.
```

Comment:
"only in addition to the status_request extension" seems unnecessarily restrictive.  Why not "only in addition to at least one of the three mechanisms listed above" ?